Kickstarter-iOS/Views/SettingsNotificationPickerCell.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+
+protocol SettingsNotificationPickerCellDelegate: class {
+  func didFailToSaveChange(errorMessage: String)
+  func didTapFrequencyPickerButton()
+  func didUpdateUser(user: User)
+}
+
+final class SettingsNotificationPickerCell: UITableViewCell, NibLoading, ValueCell {
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
+  @IBOutlet fileprivate weak var selectFrequencyButton: UIButton!
+  @IBOutlet fileprivate weak var separatorView: UIView!
+
+  weak var delegate: SettingsNotificationPickerCellDelegate?
+
+  private let viewModel: SettingsNotificationPickerViewModelType = SettingsNotificationPickerViewModel()
+
+  func configureWith(value: SettingsNotificationCellValue) {
+    self.viewModel.inputs.configure(with: value)
+
+    _ = titleLabel
+    |> UILabel.lens.text .~ value.cellType.title
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = titleLabel
+    |> settingsTitleLabelStyle
+
+    _ = selectFrequencyButton
+    |> UIButton.lens.titleLabel.font .~ .ksr_body()
+    |> UIButton.lens.titleColor(for: .normal) .~ .ksr_text_dark_grey_400
```

Comment:
Few indents in this function, just helps to see that these styles are applying to the button above. Also think we want to prefix all the property refs with `self`.